src/types/enums.ts

```diff
@@ -17,6 +17,7 @@ export enum SCRIPT_LOADING_STATE {
  */
 export enum DISPATCH_ACTION {
     LOADING_STATUS = "setLoadingStatus",
+    CRITICAL_ERROR = "criticalError",
```

Comment:
What do you think about this name? It may be nice to name it after LOADING_STATUS so it's clearer that this error message is related to a script loading failure.
```suggestion
    LOADING_STATUS_ERROR_MESSAGE = "loadingStatusErrorMessage",
```

Reply:
Good call!  Renamed